salt/modules/boto_asg.py

```
@@ -499,6 +499,59 @@ def launch_configuration_exists(name, region=None, key=None, keyid=None,
         return False
 
 
+def get_all_launch_configurations(region=None, key=None, keyid=None,
+                                  profile=None):
+    '''
+    Fetch and return all Launch Configuration with details.
+
+    CLI example::
+
+        salt myminion boto_asg.get_all_launch_configurations
+    '''
+    conn = _get_conn(region=region, key=key, keyid=keyid, profile=profile)
+    try:
+        return conn.get_all_launch_configurations()
+    except boto.exception.BotoServerError as e:
+        log.debug(e)
```

This will silently fail for most people. Should we raise this to `log.error`?